### PR TITLE
[Feature] Include quantiles in get_latest_forecast() output df 

### PR DESCRIPTION
**Pull request by judussoari**
output df of utils.fcst_df_to_last_forecast() now contains quantile columns if quantiles are specified in the model

quantile columns are generated in the same way as the yhat columns

### CODE REVIEW

File: neuralprophet/utils.py [hunk 1]
```diff
@@ -490,7 +490,7 @@ def print_epoch_metrics(metrics, val_metrics=None, e=0):
     return metrics_string
 
 
-def fcst_df_to_last_forecast(fcst, n_last=1):
+def fcst_df_to_last_forecast(fcst, quantiles, n_last=1):
     """Converts from line-per-lag to line-per-forecast.
```
Comment (ourownstory) [inline neuralprophet/utils.py:494]: please expand on the args in a docstring

File: neuralprophet/time_dataset.py [hunk 2]
```diff
@@ -383,7 +385,7 @@ def _stride_lagged_features(df_col_name, feature_dims):
             tabularized_input_shapes_str += ("    {} {} \n").format(key, value.shape)
     log.debug("Tabularized inputs shapes: \n{}".format(tabularized_input_shapes_str))
 
-    return inputs, targets, config_missing.drop_missing
+    return inputs, targets, config_missing.drop_missing, n_forecasts
```
Comment (ourownstory) [inline neuralprophet/time_dataset.py:388]: why do we need to return n_forecasts here?

File: neuralprophet/plot_forecast.py [hunk 3]
```diff
@@ -81,7 +81,7 @@ def plot(
     else:
         fig = ax.get_figure()
     ds = fcst["ds"].dt.to_pydatetime()
-    yhat_col_names = [col_name for col_name in fcst.columns if "yhat" in col_name]
+    yhat_col_names = [col_name for col_name in fcst.columns if "yhat" in col_name and "%" not in col_name]
```
Comment (ourownstory) [inline neuralprophet/plot_forecast.py:84]: will this not affect regular plotting of quantiles?

File: neuralprophet/forecaster.py [hunk 4]
```diff
@@ -720,6 +720,24 @@ def predict(self, df, decompose=True, raw=False):
         forecast = pd.DataFrame()
         for df_name, df_i in df.groupby("ID"):
             dates, predicted, components = self._predict_raw(df_i, df_name, include_components=decompose)
+            if self.config_missing.drop_missing and self.max_lags > 0:
```
Comment (ourownstory) [inline neuralprophet/forecaster.py:723]: could we make this into a function?
Why do we need to call this in predict?

File: neuralprophet/utils.py (hunk 1 above)
Comment (judussoari) [reply neuralprophet/utils.py:494]: done! Pls see PR #764 as this PR will be closed

File: neuralprophet/time_dataset.py (hunk 2 above)
Comment (judussoari) [reply neuralprophet/time_dataset.py:388]: I found a solution without returning n_forecasts. pls see PR #747 as this PR will be closed

File: neuralprophet/plot_forecast.py (hunk 3 above)
Comment (judussoari) [reply neuralprophet/plot_forecast.py:84]: You are right! Pls see PR #764 where I found a better solution

File: neuralprophet/forecaster.py (hunk 4 above)
Comment (judussoari) [reply neuralprophet/forecaster.py:723]: Done! Pls see PR #747

**Pull request closed without merging**